department-of-veterans-affairs/caseflow
# Redirect mixed case CSS_IDs in URL

Resolves #13912 

### Description
If the CSS_ID in the URL has a lowercase character, redirect to a URL using a proper uppercased CSS_ID, ensuring that other parts of the URL (e.g., path and query string) are retained.

### Acceptance Criteria
- [x] When a user provides an incorrectly cased css_id, the URL is updated to the all caps css_id
- [X] When a user provides an incorrectly cased css_id, the back end returns the correct user for the upcased css_id

### Testing Plan

For AC 1:
1. Go to http://localhost:3000/queue/BVAAAbshire/assign (see original ticket)
  - [ ] URL in browser should be corrected to http://localhost:3000/queue/BVAAABSHIRE/assign

For AC 2:
1. Go to http://localhost:3000/users?css_id=casper
  - [ ] Results shown with the correct user regardless of mixed case CSS_ID
2. Go to http://localhost:3000/user?css_id=BVAAABShire or http://localhost:3000/user?css_id=bVADHUDson
  - [ ] Result shown with the correct user regardless of mixed case CSS_ID

### User Facing Changes

Only the corrected URL is changed if CSS_ID has lowercase character(s).
